charts/furiosa-metrics-exporter/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: furiosa-metrics-exporter
 description: A Helm chart for furiosa-metrics-exporter
 type: application
-version: 0.1.2
-appVersion: "0.1.0"
+version: 0.1.3
+appVersion: "v0.0.12"
```

Comment:
any reason to update appVersion?

Reply:
Just want to match with our [latest release](https://github.com/furiosa-ai/furiosa-metrics-exporter/releases/tag/v0.0.12).

Reply:
for now we use "0.1.0" for all helm charts